src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/transfercase/TransferCaseNotificationsService.java

```diff
@@ -1,66 +1,86 @@
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
-import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.EmailTemplates;
-import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
-import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
 import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationService;
 import uk.gov.hmcts.cmc.domain.models.Claim;
 
 import java.util.HashMap;
 import java.util.Map;
 
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.CLAIMANT;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.CaseTransferred.referenceForCaseTransferred;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.DEFENDANT;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIMANT_NAME;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIM_REFERENCE_NUMBER;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.COURT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.DEFENDANT_NAME;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.EXTERNAL_ID;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.FRONTEND_BASE_URL;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.PARTY_NAME;
 
 @Service
 public class TransferCaseNotificationsService {
 
-    private final NotificationService notificationService;
-    private final NotificationsProperties notificationsProperties;
+    private static final String CCBC = "County Court Business Centre";
 
-    public TransferCaseNotificationsService(NotificationService notificationService,
-                                            NotificationsProperties notificationsProperties) {
-        this.notificationService = notificationService;
-        this.notificationsProperties = notificationsProperties;
+    @Autowired
+    private NotificationService notificationService;
+
+    @Value("${notifications.frontendBaseUrl}")
+    private String frontendBaseUrl;
+
+    @Value("${notifications.templates.email.caseTransferToCourt}")
+    private String caseTransferToCourtTemplate;
+
+    @Value("${notifications.templates.email.caseTransferToCcbcForClaimant}")
+    private String caseTransferToCcbcForClaimantTemplate;
+
+    @Value("${notifications.templates.email.caseTransferToCcbcForDefendant}")
+    private String caseTransferToCcbcForDefendantTemplate;
```

Comment:
These should be autowired via the constructor rather than by field.

Reply:
Fixed